feature/org.wso2.carbon.identity.common.feature/pom.xml

```diff
@@ -105,6 +109,10 @@
                                     <symbolicName>org.wso2.carbon.identity.event</symbolicName>
                                     <version>${project.version}</version>
```

Comment:
Shall we define a version property for identity.commons at root pom and use it here?